### PR TITLE
Filter duplicate content

### PR DESCRIPTION
**Pull request by PurpleGuitar**
This change adds a config file, `content_filters.json`, along with code in `functions.js` that defines specific language-content combinations to filter out of the list displayed on BIEL.

### CODE REVIEW

File: functions.js [hunk 1]
```diff
@@ -58,6 +58,20 @@ function mapContentLinks(languages) {
   }));
 }
 
+const contentFilters = require('./data/content_filters.json');
+function filterContents(languages) {
+    return languages.map(l => ({
+        ...l,
+        contents: l.contents.filter((c) => {
+           if (l.code in contentFilters && contentFilters[l.code].includes(c.code)) {
+               console.log('Filtered out lang:' + l.code + " content:" + c.code);
```
Comment (rbnswartz) [inline functions.js]: I don't know if this is necessarily needed but ok

**Review by rbnswartz — APPROVED** [review]
Looks good.

File: functions.js (hunk 1 above)
Comment (rbnswartz) [reply functions.js]: Mainly the console.log

Comment (PurpleGuitar) [reply functions.js]: Thanks for your note.  I was going for consistency with the rest of the code, where it logs to the console when it does something unusual.  I'm fine with taking it out though; it's basically a debug statement.

**Pull request merged**